addons/api/.eslintrc.js

```diff
@@ -11,17 +11,19 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 'latest',
     sourceType: 'module',
-    // This is specific babel-config. If grows consider creating a babel config file
```

Comment:
None blocking comment: was this comment deleted in purpose?

Reply:
I totally missed this comment, apologies! That's a good point, I had originally thought this comment was originally added by ember. I'll add it back.